apps/src/templates/readonly.html.ejs

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <title>Blockly</title>
   <link href="<%= assetUrl('css/common.css') %>" media="all" rel="stylesheet">
+  <script type="text/javascript" src="<%= assetUrl('js/manifest.js') %>"></script>
```

Comment:
Talked this over offline -- I think a better approach might be to keep `digestManifest` internal to `apps` and have `apps` own the loading of digested files.  I'll need to add configurable load hooks so project data can still be loaded at the right time.